assets/logos/logo-weesp-aligned.svg

```diff
@@ -0,0 +1,46 @@
+            <path d="M43.638,34.2154 C43.488,34.3694 43.334,34.5174 43.179,34.6544 C41.986,35.7154 40.714,36.2734 40.4,36.2734" id="Fill-37" fill="#000000"></path>
+            <path d="M31.1975,34.2154 L31.1975,35.2434 L31.1975,36.2734 L38.3425,36.2734 L40.4005,36.2734 C40.7145,36.2734 41.9865,35.7154 43.1785,34.6544 C43.3345,34.5174 43.4875,34.3694 43.6375,34.2154 L38.3425,34.2154 L31.1975,34.2154 Z" id="Fill-39" fill="#002B7F"></path>
+            <path d="M31.1975,30.0132 L31.1975,31.0432 L31.1975,32.0722 L38.3425,32.0722 L45.0775,32.0722 C45.3325,31.4502 45.4875,30.7632 45.4875,30.0132 L38.3425,30.0132 L31.1975,30.0132 Z" id="Fill-41" fill="#002B7F"></path>
+            <polygon id="Fill-43" fill="#002B7F" points="0.412 2.664 14.702 2.664 14.702 0.606 0.412 0.606"></polygon>
+            <polygon id="Fill-45" fill="#002B7F" points="0.412 6.865 14.702 6.865 14.702 4.807 0.412 4.807"></polygon>
+            <polygon id="Fill-46" fill="#002B7F" points="0.412 11.066 14.702 11.066 14.702 9.008 0.412 9.008"></polygon>
+            <polygon id="Fill-47" fill="#002B7F" points="0.412 15.267 14.702 15.267 14.702 13.209 0.412 13.209"></polygon>
+            <polygon id="Fill-48" fill="#002B7F" points="0.412 19.469 14.702 19.469 14.702 17.411 0.412 17.411"></polygon>
+            <polygon id="Fill-49" fill="#002B7F" points="0.412 23.669 14.702 23.669 14.702 21.611 0.412 21.611"></polygon>
+            <polygon id="Fill-50" fill="#002B7F" points="0.4124 27.8716 7.5574 27.8716 14.7024 27.8716 14.7024 26.8426 14.7024 25.8126 7.5574 25.8126 0.4124 25.8126 0.4124 26.8426"></polygon>
+            <path d="M2.2615,34.2154 C2.4115,34.3694 2.5655,34.5174 2.7195,34.6544 C3.9135,35.7154 5.1845,36.2734 5.4995,36.2734" id="Fill-51" fill="#000000"></path>
+            <path d="M14.7024,34.2154 L14.7024,35.2434 L14.7024,36.2734 L7.5574,36.2734 L5.4994,36.2734 C5.1844,36.2734 3.9134,35.7154 2.7204,34.6544 C2.5654,34.5174 2.4114,34.3694 2.2614,34.2154 L7.5574,34.2154 L14.7024,34.2154 Z" id="Fill-52" fill="#002B7F"></path>
+            <path d="M14.7024,30.0132 L14.7024,31.0432 L14.7024,32.0722 L7.5574,32.0722 L0.8214,32.0722 C0.5664,31.4502 0.4124,30.7632 0.4124,30.0132 L7.5574,30.0132 L14.7024,30.0132 Z" id="Fill-53" fill="#002B7F"></path>
+            <path d="M22.9666,39.815 C23.2246,38.425 25.3966,35.828 29.4346,35.828 L29.4346,1.044 L22.8736,1.044 L22.8426,1.044 L16.4726,1.044 L16.4726,35.828 C20.5086,35.828 22.6806,38.423 22.9396,39.815 L22.9526,39.865 L22.9666,39.815 Z" id="Stroke-54" stroke="#002B7F" stroke-width="0.914"></path>
+            <path d="M0.4124,44.5464 C2.9374,42.6364 5.4254,42.1804 7.9044,42.1804 C10.6034,42.1804 11.5954,42.3984 15.4514,44.3634 C19.3074,46.3294 21.4434,46.2764 22.9094,46.2764 C24.3754,46.2764 26.4574,46.2024 30.3674,44.3144 C34.3904,42.3694 35.5634,42.1804 38.0924,42.1804 C40.5704,42.1804 42.9624,42.6364 45.4554,44.5464" id="Stroke-55" stroke="#002B7F" stroke-width="0.915"></path>
+        </g>
+    </g>
+</svg>
```

Comment:
This file can be minified and its file size can be reduced roughly by half.

Reply:
> Please minify the SVG file

I assume we have HTTP compression enabled, so the real data gain will be minimal while we obfuscate something readable.

If minification is still desired I would propose to make a minified copy a next to the `original` and use that if you agree?